task_schedule_check_cron.cfg

```diff
@@ -0,0 +1,30 @@
+# Configuration file for task_schedule.pl to do a daily check of the kadi cron jobs.
+
+subject           Kadi check cron jobs
+timeout           3000              # Default tool timeout
+heartbeat_timeout 10                # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
```

Comment:
It seems like the other option would have been to make the events job a persistent cron, but if we don't like those anymore, this makes sense.